src/App.js

```diff
@@ -1,10 +1,176 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css';
+import logo from './assets/screenshot-logo.png'
+import FruitButtons from "./FruitButtons";
+import {fireEvent} from "@testing-library/react";
+import FormEntry from "./FormEntry";
 
 function App() {
+
+
+    const [fruitState, setFruitState] = useState( {
+        strawberries : 0,
+        bananas : 0,
+        apples : 0,
+        kiwis : 0
+    })
+
+    const [formState, setFormState] = useState({
+        firstname : '',
+        lastname: '',
+        age : 0,
+        postalCode : '',
+        frequency : 'Iedere week',
+        moment : 'daytime',
+        comment : '',
+        conditions : false,
+    })
+
+
+
+    function handleChange(e) {
+        const changedFieldName = e.target.name;
```

Comment:
Hier kun je spreaden, zodat je niet overal `e.target.` vóór hoeft te zetten
```suggestion
    function handleChange({target: { type, checked, value, name: changedFieldName }}) {
```